src/dataframe.js

```diff
@@ -775,9 +775,10 @@ class DataFrame {
      * @example
```

Comment:
Just add the description of the new parameter columnNames (to show it in the documentation).
Then, it's fine :D